app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -56,7 +57,20 @@ class WallpaperSettingsFragment : Fragment() {
                     val coroutineScope = rememberCoroutineScope()
 
                     WallpaperSettings(
-                        wallpaperGroups = wallpapers.groupByDisplayableCollection(),
+                        wallpaperGroups = if (FeatureFlags.wallpaperV2Enabled) {
+                            wallpapers.groupByDisplayableCollection()
+                        } else {
+                            val firefoxClassicCollection = Wallpaper.Collection(
+                                name = "classic-firefox",
```

Comment:
I've seen the `"classic-firefox"` string in a few places now. Would it make sense to put this as a const val somewhere?

Reply:
I have also seen this enough that I think we should probably make this a constant that lives somewhere.

Reply:
Yeah, went ahead and added the collection and the name locally. It gives me the ick a bit since I would like to be able to rely on the data being dynamic, but I don't immediately see another good way to handle these edge cases. Might be worth revisiting in the future